test/Microsoft.Buld.Sql.Tests/ProjectUtils.cs

```diff
@@ -28,7 +51,10 @@ public static void AddItemGroup(string projectFilePath, string itemName, string[
                     ProjectItemGroupElement itemGroup = project.Xml.AddItemGroup();
                     foreach (string filePath in filePaths)
                     {
-                        itemGroup.AddItem(itemName, filePath);
+                        ProjectItemElement item = project.Xml.CreateItemElement(itemName);
```

Comment:
`itemGroup.AddItem()` should return the new item, if you wanted to make this a little more concise.